test/dynamo/test_misc.py

```diff
@@ -1680,24 +1680,48 @@ def fn(packed):
         v3 = torch.Tensor([3])
         cnts = torch._dynamo.testing.CompileCounter()
         opt_fn = torch._dynamo.optimize(cnts)(fn)
-        self.assertEqual(opt_fn(mytuple(v1, v2, v3))[0], 7)
+        self.assertEqual(opt_fn(MyTuple(v1, v2, v3))[0], 7)
         self.assertEqual(cnts.frame_count, 1)
         self.assertEqual(cnts.op_count, 3)
 
     def test_namedtuple3(self):
         def fn(x, packed):
-            if isinstance(packed, mytuple):
+            if isinstance(packed, MyTuple):
                 return x + 1
             else:
                 return x - 1
 
         x = torch.rand([2, 3])
-        packed = mytuple(1, 2, 3)
+        packed = MyTuple(1, 2, 3)
         ref = fn(x, packed)
         opt_fn = torch._dynamo.optimize("eager")(fn)
         res = opt_fn(x, packed)
         self.assertTrue(same(ref, res))
 
+    def test_structseq1(self):
+        def fn(x, y):
+            return torch.return_types.max((x, y))
+
+        x = torch.randn(3, 2)
+        y = torch.randn(2, 4)
+        expected = fn(x, y)
+        fn_opt = torch.compile(fullgraph=True)(fn)
+        actual = fn_opt(x, y)
+
+        self.assertEqual(actual, expected)
+
+    def test_structseq2(self):
+        def fn(x, y):
+            return tuple(torch.return_types.qr((2 * x, y - 1)))
```

Comment:
I have no idea about why this is falling. The traced function seems correct:

```console
$ export TORCHDYNAMO_VERBOSE=1
$ TORCH_LOGS="+dynamo"
$ python3 test/dynamo/test_misc.py -k structseq
...
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code] TRACED GRAPH
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]  ===== __compiled_fn_2 =====
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]  /Users/PanXuehai/Projects/pytorch/torch/fx/_lazy_graph_module.py class GraphModule(torch.nn.Module):
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]     def forward(self, L_x_: "f32[3, 2][2, 1]cpu", L_y_: "f32[2, 4][4, 1]cpu"):
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]         l_x_ = L_x_
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]         l_y_ = L_y_
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]         
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]          # File: /Users/PanXuehai/Projects/pytorch/test/dynamo/test_misc.py:1715 in fn, code: return tuple(torch.return_types.qr((2 * x, y - 1)))
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]         mul: "f32[3, 2][2, 1]cpu" = 2 * l_x_;  l_x_ = None
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]         sub: "f32[2, 4][4, 1]cpu" = l_y_ - 1;  l_y_ = None
V1013 00:13:14.346000 79432 torch/_dynamo/output_graph.py:1341] [0/0] [__graph_code]         return (mul, sub)
```

i.e.:

```python
def forward(x, y):
    mul = 2 * x
    sub = y - 1
    return (mul, sub)
```

But the compiled function returns:

```python
def forward(x, y):
    mul = 2 * x
    sub = y - 1
    return ((mul, sub), sub)
```

cc @jansel @anijain2305 for help.

Reply:
Resolved by update `UserDefinedClassVariable(structseq_type).call_function(...)`.